GeoLlama/calc_by_slice.py

```diff
@@ -280,41 +269,43 @@ def evaluate_slice(slice_coords: list,
     mask_s1 = np.argwhere(view**2 >= thres)
```

Comment:
why are the pixel values being squared?

Reply:
I could've just used the magic number 1.12 times the threshold (I've mentioned it in the manuscript) - I'll change it in the code.